projects/ion/src/lib/checkbox/checkbox.component.html

```diff
@@ -1,7 +1,15 @@
-<input
-  data-testid="ion-checkbox"
-  class="ion-checkbox"
-  type="checkbox"
-  (click)="changeState()"
-  #checkBox
-/>
+<div class="ion-input-checkbox">
+  <input
```

Comment:
You need to have the same value in the `id` of the input and the `for` of the label, so that when you click on label, the checkbox will be selected/unselected ([more here](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/Input/checkbox#fornecendo_uma_%C3%A1rea_de_acertos_maior_para_suas_caixas_de_sele%C3%A7%C3%A3o)).

Reply:
thanks for the tip